core/src/main/scala/org/apache/spark/storage/RDDInfo.scala

```diff
@@ -53,10 +55,16 @@ class RDDInfo(
 }
 
 private[spark] object RDDInfo {
+  private val callsiteForm = SparkEnv.get.conf.get(EVENT_LOG_CALLSITE_FORM)
+
   def fromRdd(rdd: RDD[_]): RDDInfo = {
     val rddName = Option(rdd.name).getOrElse(Utils.getFormattedClassName(rdd))
     val parentIds = rdd.dependencies.map(_.rdd.id)
+    val callSite = callsiteForm match {
+      case "short" => rdd.creationSite.shortForm
+      case "long" => rdd.creationSite.longForm
```

Comment:
If the users input is neither `short` nor `long`, we will get an exception, right?

cc @jiangxb1987 @cloud-fan

Reply:
yea, usually we will define an enum and verify the given config value is valid or not.

For this particular case, I think we can just define a boolean flag: `spark.eventLog.callsite.longForm.enabled`.